docker/docker-compose.client.dev.yml

```diff
@@ -3,9 +3,6 @@ version: '3.5'
 # bindings.  See the readme for more information.
 services:
   client:
-    build:
```

Comment:
Without this, I believe local changes on the client dockerfile wouldn't be reflected? The `-d` flag means we always want to build from the local dockerfile instead of using the image

Reply:
You need to choose. Either we want to simplify user to always have a working version of client without the need to build from a known commit, or they use our provided container.

Reply:
Sorry I'm not getting it. As it is, we can use the `-d` flag to build and use the container, as it is in the current dir, and we bind mount and (I think) use `nodemon` to restart with changes. Without the `-d` flag it pulls the client from somewhere (in this case, AWS) and uses that.

Not sure if this `-d` flag could be added to `start-client` in the very same way

Reply:
I think you missed the purpose of the PR. The idea is that the client container is not built on the spot anymore. It is always downloaded from AWS. The benefits of this is that now you have always a synced version of the client with testnet,

Reply:
I think I get it, we're assuming someone working on the client would be using the `./start-nightfall` script, and use this docker-compose file only for pulling and using the image from AWS. I'm ok with this then 😊